roles/robotnofollow.xml

```diff
@@ -0,0 +1,531 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<role>
+    <shortname>robotnofollow</shortname>
+    <name>Robot nofollow</name>
+    <description>&lt;p&gt;This role is for Linkchecker robot plugin. It prohibits crawler from entering the courses.&lt;/p&gt;</description>
+    <archetype/>
+    <contextlevels>
+        <level>coursecat</level>
+        <level>course</level>
+        <level>module</level>
+        <level>block</level>
+    </contextlevels>
+    <allowassign>
```

Comment:
just for flexibility, cause we mention in Readme that the roles could be applied to things other than categories, like courses, activities and blocks.